Sources/Types.swift

```diff
@@ -11,8 +11,8 @@ public struct DictionaryType: Type_ {
     public var valueType: Type_
 }
 
-public struct FunctionType­: Type_ {
-    public var arguments: [(String?, Type_)]
+public struct FunctionType: Type_ {
+    public var arguments: [Type_]
```

Comment:
Yeah, see [SE-0111](https://github.com/apple/swift-evolution/blob/master/proposals/0111-remove-arg-label-type-significance.md) for the rationale.

:flushed: `testTypeSuffix`